fullstack/README.md

```diff
@@ -0,0 +1,64 @@
+This is a coding exercise to help assess candidates looking to join our team at Condé Nast International as FullStack Developers.
+
+The test is intended to be completed at home in no more than 1-2 hours. It is not timed, so if you want to take more time to showcase your coding skills and ability you may do so (note this is not mandatory, nor expected).
```

Comment:
To me this reads "you should take longer on it". I'd rather explicitly say that we don't want them to do that. Perhaps we could rephrase to:
"The exercise is intended to be completed at your leisure in around 1-2 hours. It is not timed, so if you finish early or late that's fine too. We aren't looking for perfection, just hoping to get an idea of the way you work."

Reply:
:+1: